From b980352022314102cec562f1bb21768e65628b38 Mon Sep 17 00:00:00 2001
From: David Benjamin <davidben@google.com>
Date: Tue, 7 Feb 2023 15:55:48 -0500
Subject: [PATCH] Remove stale TODO in util/bot/DEPS

We are using the CIPD copy of CMake on Windows now.

Change-Id: Idb9f62876c69333f9504540bad8321a173eaec3e
Reviewed-on: https://boringssl-review.googlesource.com/c/boringssl/+/56988
Commit-Queue: Bob Beck <bbe@google.com>
Reviewed-by: Bob Beck <bbe@google.com>
Auto-Submit: David Benjamin <davidben@google.com>
(cherry picked from commit 49d7b2d6a4d586e2debfa847e81b5ad9a9d3218a)
---
 util/bot/DEPS | 1 -
 1 file changed, 1 deletion(-)

diff --git a/util/bot/DEPS b/util/bot/DEPS
index 7f91ca971..66b4006a4 100644
--- a/util/bot/DEPS
+++ b/util/bot/DEPS
@@ -121,7 +121,6 @@ recursedeps = [
 ]
 
 hooks = [
-  # TODO(https://crbug.com/1180257): Use CIPD for CMake on Windows.
   {
     'name': 'perl_win32',
     'pattern': '.',
